### PR TITLE
Relative humidity

### LINKED ISSUE #606: Algorithm to calculate relatively humidity.
**Is your feature request related to a problem? Please describe.**

Given a temperature and a dew point calculate relative humidity.

**Describe the solution you'd like**
A simple algorithm

**Additional context**

`
To calculate relative humidity with temperature and dew point, follow the given instructions:
1.	Measure the air temperature T, in °C.
2.	Find out the dew point temperature Dp, in °C.
3.	Calculate relative humidity RH using the formula, `RH = 100 × {exp[17.625 × Dp/(243.04 + Dp)]/exp[17.625 × T/(243.04 + T)]}`.

### PR DESCRIPTION
**Pull request by oskarhurst**
## Problem Description

Fixes #606 

Describe the problem you are trying to solve.

## Solution

Created the RelativeHumidity algorithm

## how you tested the change

I created test files and confirmed the output from the algorithm was the expected data.

## Where the following done:

- [ ] Tests. Check all that apply:
   - [ ] Unit tests created or modified that run during ant test.
   - [x] Integration tests created or modified that run during integration testing
         (Formerly called regression tests.)
   - [ ] Test procedure descriptions for manual testing
- [ x] Was relevant documentation updated?
- [ ] Were relevant config element (e.g. XML data) updated as appropriate

If you aren't sure leave unchecked and we will help guide you to want needs changing where.

### CODE REVIEW

File: src/main/java/decodes/tsdb/algo/RelativeHumidity.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import decodes.tsdb.DbCompException;
+import decodes.tsdb.IntervalCodes;
+import ilex.var.NamedVariable;
+import opendcs.opentsdb.Interval;
+import org.opendcs.annotations.algorithm.Algorithm;
+import org.opendcs.annotations.algorithm.Input;
+import org.opendcs.annotations.algorithm.Output;
+
+@Algorithm(
+		description ="Takes two inputs, Air temperature and dew point, to calculate relative humidity" )
```
Comment (MikeNeilson) [inline src/main/java/decodes/tsdb/algo/RelativeHumidity.java:12]: Description should just start right after the (, there doesn't need to be a new line here.

File: src/main/java/decodes/tsdb/algo/RelativeHumidity.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+public class RelativeHumidity
+	extends AW_AlgorithmBase
```
Comment (MikeNeilson) [inline src/main/java/decodes/tsdb/algo/RelativeHumidity.java:14]: extends ... should be on the same line.

File: src/main/java/decodes/tsdb/algo/RelativeHumidity.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+	public NamedVariable output = new NamedVariable("output", 0);
+
+
+	// Allow javac to generate a no-args constructor.
+
+	/**
+	 * Algorithm-specific initialization provided by the subclass.
+	 */
+	protected void initAWAlgorithm( )
+		throws DbCompException
```
Comment (MikeNeilson) [inline src/main/java/decodes/tsdb/algo/RelativeHumidity.java:32]: Put all the throws on the same line after the method.

File: src/main/java/decodes/tsdb/algo/RelativeHumidity.java [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+	 * User code should call one of the setOutput methods for a time-slice
+	 * output variable.
+	 *
+	 * @throws DbCompException (or subclass thereof) if execution of this
+	 *        algorithm is to be aborted.
+	 */
+	protected void doAWTimeSlice()
+		throws DbCompException
+	{
+		if(isMissing(temperature) || isMissing(dewPoint)){
```
Comment (MikeNeilson) [inline src/main/java/decodes/tsdb/algo/RelativeHumidity.java:77]: brackets on next line, and space between if and (.

**Review by MikeNeilson — CHANGES_REQUESTED** [review]
Logic looks good. Just some nitpicky formatting changes that the existing computations are just a bad example of.

File: src/main/java/decodes/tsdb/algo/RelativeHumidity.java (hunk 3 above)
Comment (oskarhurst) [reply src/main/java/decodes/tsdb/algo/RelativeHumidity.java]: I was use copying the original style but I made all recommended changes, thank you for reviewing it!

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**